test/relations.test.js

```diff
@@ -159,12 +162,20 @@ describe('relations', function() {
             var chapters = book.chapters();
             chapters.should.eql(ch);
 
-            book.chapters({order: 'name DESC'}, function(e, c) {
+            var nameFilter = isCassandraConnector ? {} : {order: 'name DESC'};
```

Comment:
If `containEql` does not work, then you can try `containDeep`. I am not sure about the exact differences between these two methods.